app/pdf.js

```diff
@@ -5,7 +5,7 @@
 'use strict'
 
 const tabs = require('./browser/tabs')
-const {fileUrl} = require('../../js/lib/appUrlUtil')
+const {fileUrl} = require('../js/lib/appUrlUtil')
```

Comment:
@petemill note that this part doesn't apply to master (the bug doesn't exist in master at all). haven't checked 0.21.x.